src/resources/help/en_US/relnotes3.1.3.html

```diff
@@ -52,7 +52,8 @@
   <li>The Lineout operator documentation was improved in the GUI manual to describe in detail how the sampling and interpolation is performed.</li>
   <li>Added the ability to generate ghost nodes for datasets that consist of collections of structured grids by matching the coordinates of the nodes on the exterior of individual structured grids. The coordinates must match exactly for them to be matched. Ghost nodes allow internal faces to be eliminated from Mesh, Filled Boundary, Pseudocolor and Subset plots. This improves rendering performance by reducing the amount of geometry to render. It also improves image quality for images rendered with transparency since it eliminates geometry that you are most likely not interested in seeing.</li>
   <li>Updated the host profiles for the Sandia National Laboratories computer systems, adding host profiles for new systems and removing ones for obsolete systems.</li>
-  <li>Added support for the FILETYPE keyword to the Tecplot reader. The FULL type is the only type supported. If the FILETYPE keyword is specified and the type isn't FULL, an error is generated.</li>
+  <li>Added support for the FILETYPE keyword to the Tecplot reader. The FULL type is the only type supported. An error is generated if it isn't.</li>
```

Comment:
I changed the comment about the FILETYPE after you branched, so you should take what is on develop for it.